client/src/components/Tool/ToolCard.vue

```diff
@@ -26,6 +26,9 @@
                         <b-dropdown-item v-if="showDownload" @click="onDownload"
                             ><span class="fa fa-download" /><span v-localize>Download</span>
                         </b-dropdown-item>
+                        <b-dropdown-item v-b-modal.tool-source-viewer
+                            ><span class="fa fa-eye" /><span v-localize>Preview tool source</span>
```

Comment:
How about `View Tool source` or `View Tool XML`  in order to align it with `View Tool ID`?